lib/content-services/src/lib/search/services/base-query-builder.service.ts

```diff
@@ -73,8 +73,14 @@ export abstract class BaseQueryBuilderService {
     /*  Stream that emits the initial value for some or all search filters */
     populateFilters = new BehaviorSubject<{ [key: string]: any }>({});
 
+    /*  Stream that emits every time queryFragments change */
+    queryFragmentsUpdate = new BehaviorSubject<{ [key: string]: any }>({});
```

Comment:
here we should have just string instead of any because it emits queryFragments in next function and queryFragments has string there instead of any